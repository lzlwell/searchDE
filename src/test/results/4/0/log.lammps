LAMMPS (30 Aug 2012)
units		metal
boundary	p p p
atom_style	atomic
read_data	struct.lammps
  orthogonal box = (0 0 0) to (17.1317 10 50)
  1 by 1 by 1 MPI processor grid
  300 atoms

pair_style      airebo 3.0
pair_coeff      * * CH.airebo C H

mass	1	12.0
mass	2	1.0

group		fixPosH id <> 1 120
120 atoms in group fixPosH
group		fixPosL id <> 121 239
119 atoms in group fixPosL
group		optPos  id <> 240 1000
61 atoms in group optPos
fix             0       fixPosH setforce NULL 0.0 NULL
fix             1       fixPosH aveforce 0.0 0.0 0.0
fix		2	fixPosL setforce 0.0 0.0 0.0
fix 		3	optPos	setforce NULL 0.0 NULL

neighbor        0.2 bin
neigh_modify	every 1 delay 0 check yes

timestep	0.002
thermo 		1
thermo_style    custom step  pe etotal lx ly lz 

dump		1 all xyz 10000000 dump.atom
dump_modify	1 format "%d  %20.15g  %20.15g  %20.15g" 
#fix 		8 all box/relax iso 0.0 
min_style	cg
minimize	1.0e-12 1.0e-12 10 10
Memory usage per processor = 3.26712 Mbytes
Step PotEng TotEng Lx Ly Lz 
       0     -130.321     -130.321      17.1317           10           50 
       1   -796.70955   -796.70955      17.1317           10           50 
       2   -972.77904   -972.77904      17.1317           10           50 
       3   -1036.5368   -1036.5368      17.1317           10           50 
       4   -1070.3491   -1070.3491      17.1317           10           50 
       5   -1092.0312   -1092.0312      17.1317           10           50 
       6   -1116.1963   -1116.1963      17.1317           10           50 
       7   -1135.4708   -1135.4708      17.1317           10           50 
       8   -1147.7052   -1147.7052      17.1317           10           50 
       9   -1154.7341   -1154.7341      17.1317           10           50 
      10    -1159.274    -1159.274      17.1317           10           50 
Loop time of 0.359896 on 1 procs for 10 steps with 300 atoms

Minimization stats:
  Stopping criterion = max force evaluations
  Energy initial, next-to-last, final = 
        -130.321000399     -1154.73406276     -1159.27404345
  Force two-norm initial, final = 8583.04 24.5661
  Force max component initial, final = 5835.37 10.2507
  Final line search alpha, max atom move = 0.00541076 0.0554642
  Iterations, force evaluations = 10 10

Pair  time (%) = 0.140334 (38.9929)
Neigh time (%) = 0.216089 (60.042)
Comm  time (%) = 0.00144625 (0.401851)
Outpt time (%) = 0.000141144 (0.0392179)
Other time (%) = 0.00188589 (0.52401)

Nlocal:    300 ave 300 max 300 min
Histogram: 1 0 0 0 0 0 0 0 0 0
Nghost:    2357 ave 2357 max 2357 min
Histogram: 1 0 0 0 0 0 0 0 0 0
Neighs:    0 ave 0 max 0 min
Histogram: 1 0 0 0 0 0 0 0 0 0
FullNghs:  46810 ave 46810 max 46810 min
Histogram: 1 0 0 0 0 0 0 0 0 0

Total # of neighbors = 46810
Ave neighs/atom = 156.033
Neighbor list builds = 10
Dangerous builds = 10
min_modify line quadratic

clear
