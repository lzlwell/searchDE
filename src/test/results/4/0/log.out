LAMMPS (30 Aug 2012)
Reading data file ...
  orthogonal box = (0 0 0) to (17.1317 10 50)
  1 by 1 by 1 MPI processor grid
  300 atoms
120 atoms in group fixPosH
119 atoms in group fixPosL
61 atoms in group optPos
Setting up minimization ...
Memory usage per processor = 3.26712 Mbytes
Step PotEng TotEng Lx Ly Lz 
       0     -130.321     -130.321      17.1317           10           50 
       1   -796.70955   -796.70955      17.1317           10           50 
       2   -972.77904   -972.77904      17.1317           10           50 
       3   -1036.5368   -1036.5368      17.1317           10           50 
       4   -1070.3491   -1070.3491      17.1317           10           50 
       5   -1092.0312   -1092.0312      17.1317           10           50 
       6   -1116.1963   -1116.1963      17.1317           10           50 
       7   -1135.4708   -1135.4708      17.1317           10           50 
       8   -1147.7052   -1147.7052      17.1317           10           50 
       9   -1154.7341   -1154.7341      17.1317           10           50 
      10    -1159.274    -1159.274      17.1317           10           50 
Loop time of 0.359896 on 1 procs for 10 steps with 300 atoms

Minimization stats:
  Stopping criterion = max force evaluations
  Energy initial, next-to-last, final = 
        -130.321000399     -1154.73406276     -1159.27404345
  Force two-norm initial, final = 8583.04 24.5661
  Force max component initial, final = 5835.37 10.2507
  Final line search alpha, max atom move = 0.00541076 0.0554642
  Iterations, force evaluations = 10 10

Pair  time (%) = 0.140334 (38.9929)
Neigh time (%) = 0.216089 (60.042)
Comm  time (%) = 0.00144625 (0.401851)
Outpt time (%) = 0.000141144 (0.0392179)
Other time (%) = 0.00188589 (0.52401)

Nlocal:    300 ave 300 max 300 min
Histogram: 1 0 0 0 0 0 0 0 0 0
Nghost:    2357 ave 2357 max 2357 min
Histogram: 1 0 0 0 0 0 0 0 0 0
Neighs:    0 ave 0 max 0 min
Histogram: 1 0 0 0 0 0 0 0 0 0
FullNghs:  46810 ave 46810 max 46810 min
Histogram: 1 0 0 0 0 0 0 0 0 0

Total # of neighbors = 46810
Ave neighs/atom = 156.033
Neighbor list builds = 10
Dangerous builds = 10
